lib/countly.js

```diff
@@ -56,6 +56,12 @@
 }(typeof window !== "undefined" ? window : this, function(Countly) {
     "use strict";
 
+    // Make sure the code is being run in a browser
+    // eslint-disable-next-line no-undef
+    if (!process.browser) {
```

Comment:
I think this will give a ReferenceError for process when this sdk script will be run in browser.
process will be undefined in the browser.

Reply:
https://stackoverflow.com/questions/49411796/how-do-i-detect-whether-i-am-on-server-on-client-in-next-js

Reply:
Ahh I see, it's gonna work in React/Next.js apps but not in vanilla js apps. 

Let me check if this condition is gonna do the job for next js as well ;)

```
typeof(window) != 'undefined'
```